core/templates/dev/head/pages/exploration_editor/improvements_tab/PlaythroughImprovementCardDirective.js

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+oppia.directive('playthroughImprovementCard', [
+  'UrlInterpolationService', function(UrlInterpolationService) {
+    return {
+      restrict: 'E',
+      scope: {
+        data: '=',
```

Comment:
Does this need to be bidierectional? If not consider '&' instead. `getData: '&data'`

Reply:
Done.

Reply:
@seanlip just an FYI before I mark this as resolved.

Reply:
I think & should work if it's a one-way read -- this probably needs some debugging. Maybe take a look at how it's used elsewhere in the codebase?

Reply:
You may need to replace "data" with "getData()" in the appropriate places.